examples/ipc_ipcapi/ipc_ipcapi_anon_fd.sh

```diff
@@ -12,21 +12,6 @@ set -e
 # port should be a number from the range <1024, 65535>
 PORT=$(( 1024 + ( $$ % ( 65535 - 1024 ))))
 
```

Comment:
it's this time of the year - pls, bump license(s)

Reply:
Thanks, done